assets/js/state/selectors/lastExecutions.test.js

```diff
@@ -117,18 +188,24 @@ describe('lastExecutions selector', () => {
       },
       catalog: {},
       lastExecutions: {
-        [clusterID]: {
+        [groupID]: {
           loading: false,
           data: runningExecution,
           error: null,
         },
       },
     };
 
-    const { lastExecution } = getLastExecutionData(state, clusterID);
+    ['cluster', 'host'].forEach(({ targetType }) => {
```

Comment:
Does this destructuring assignment work? :raised_eyebrow: Does `targetType` get assgined?

Reply:
Test still passes because targetType has no impact in the scenario